src/sep6/types.d.ts

```diff
@@ -0,0 +1,205 @@
+export type getKycUrl = (args: GetKycArgs) => string;
+
+export interface GetKycArgs {
+  request: WithdrawRequest | DepositRequest;
+  response: InteractiveKycNeeded;
+  callbackUrl: string;
+}
+
+export interface FeeArgs {
+  assetCode: string;
+  amount: string;
+  operation: "withdraw" | "deposit";
+  type: string;
+}
+
+export interface RawInfo {
```

Comment:
I think this type name could be more descriptive, this seems like "TransferInfoResponse".

Reply:
I like `Raw` to signify that it's unprocessed, but yeah I'll make it a little more verbose.